packages/component-library/src/BaseMap/BaseMap.js

```diff
@@ -32,6 +32,7 @@ class BaseMap extends Component {
         maxZoom: 16,
         pitch: props.initialPitch || 0,
         bearing: 0,
+        scrollZoom: false,
```

Comment:
On Chrome on MacOS using a trackpad, not having scrollZoom feels inconsistent with the behavior of other interactive maps. If this is targeting other use cases (e.g. mobile), maybe we can selectively target those use cases.

Reply:
I'll figure this out in a future iteration of the Base Map. I changed it back since it's not a necessary change for the dashboard.